packages/catana/PKGBUILD

```diff
@@ -0,0 +1,27 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=catana
+pkgver=1.0
+pkgrel=1
+pkgdesc='CATANA filters your wordlist according to the specified password policy.'
+groups=('blackarch' 'blackarch-misc')
+arch=('any')
+url='https://github.com/D3vil0per/catana'
+license=('GNU General Public License v3.0')
+depends=('bash')
+makedepends=('git')
+source=("git+https://github.com/D3vil0per/$pkgname.git")
+
+pkgver() {
+	cd $pkgname
```

Comment:
you have mixed indentation.

Reply:
Commit 9090a8bfaf716fabb0923bcf318e2de25821a587